.gitlab-ci.yml

```diff
@@ -137,10 +137,14 @@ TF_Service::dotNet::build:
   stage: build
   when: always
   script:
-    # Move the built wrapper to the root directory
-    - dotnet publish -c Release "$CI_PROJECT_DIR/TF_Service_dotNet/TouchFree Service/TouchFree Service.sln"
+    - dotnet build -c Release "$CI_PROJECT_DIR/TF_Service_dotNet/TouchFree Service/TouchFree Service.sln"
+
+    # Run unit tests
+    - dotnet test "$CI_PROJECT_DIR/TF_Service_dotNet/TouchFreeTests/TouchFreeTests.csproj"
+
+    # Move the built executable to the root directory
     - mkdir ./Service_dotNet_Build
-    - cp -r "$CI_PROJECT_DIR/TF_Service_dotNet/TouchFree Service/bin/Release/net5.0/publish/*" ./Service_dotNet_Build
+    - cp -r "$CI_PROJECT_DIR/TF_Service_dotNet/TouchFree Service/bin/Release/net5.0/*" ./Service_dotNet_Build
```

Comment:
I believe @MattGrayUL was publishing the build so it included a .NET5 runtime, did we figure out another way around that?